<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineOptionWidget</class>
 <widget class="QWidget" name="EngineOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EngineOptionWidget</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>591</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout" rowstretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QGroupBox" name="engineGroup">
        <property name="title">
         <string>번역 엔진 설정</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="engineLayoutWidget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>20</number>
            </property>
            <property name="verticalSpacing">
             <number>16</number>
            </property>
            <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignVCenter">
             <widget class="QLineEdit" name="apiInputLine">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignVCenter">
             <widget class="QLabel" name="enginNamelabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>번역 엔진:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
             <widget class="QLabel" name="apiNamelabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>API: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
             <widget class="DropdownMenu" name="enginSelectCombo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
             <widget class="QPushButton" name="apiKeySaveButton">
              <property name="text">
               <string>저장</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropdownMenu</class>
   <extends>QComboBox</extends>
   <header>SubWidgets/DropdownMenu.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
